src/send/Send.tsx

```diff
@@ -122,6 +122,7 @@ function Send({ route }: Props) {
       return
     }
 
+    Keyboard.dismiss()
```

Comment:
what do you think about moving this to line 119, so that the keyboard is dismissed as soon as we know the verification status? the benefit would be, the keyboard is also dismissed when the invite modal appears. sorry, this is an existing issue

Reply:
ohh, interesting -- yeah, that sounds good to me, two birds one stone etc. etc. updated!